game.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+var output = document.getElementById('output');
+var paper = document.getElementById('paper');
+var scissors = document.getElementById('scissors');
+var rock = document.getElementById('rock');
+var newGame = document.getElementById('new-game');
+var playerResult = document.getElementById('player-score');
+var computerResult = document.getElementById('computer-score');
+var maxRounds;
+var currentRound = 0;
+var playerScore = 0;
+var computerScore = 0;
+var playerChoice;
+var computerChoice;
+var winner;
+
+newGame.addEventListener('click', function gameInit() {
```

Comment:
jesli juz robisz z tej `gameInit` nie anonimową funkcję to wyciągnij ją na zewnątrz, a tu się tylko do niej odwołaj:
`newGame.addEventListener('click', gameInit)`;